application/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+    id 'org.springframework.boot' version '2.7.8'
```

Comment:
What do you think about adding spotless later on?

Reply:
Didn't know that this is even a question.